### ISSUE TITLE
Provide an overview of the currently pending jobs

### ISSUE THREAD

**Issue by metaodi** [davidread/ckanext-xloader#37]
We just deployed xloader on a production server. And since we have harvesters running that fill up the pipeline and users that are manually triggering an upload it would be very nice to have some kind of overview how many jobs are currently pending and which are being processed.

**Comment by davidread** [comment]
Agreed, it's all rather opaque. What are you thinking? A command-line tool / api / web page?

There's ``paster --plugin=ckan jobs list`` for pending jobs. When a job starts, the status is tracked in the ``xloader_jobs`` database (see readme).

Note that the separate db is just there because legacy - that's how datapusher did it. As it says in the readme: "Job results are currently still stored in its own database, but the intention is to move this relatively small amount of data into CKAN's database, to reduce the complication of install." So if you were particularly keen and this is something you fancied doing too, that'd be a help.
